app/server/api.py

```diff
@@ -1,19 +1,57 @@
-from fastapi import FastAPI
+from fastapi import FastAPI, HTTPException, status
 
-from app.schemas.rules import RequestObject
+from app.database.policy_database import add_policy, update_policy, exists, get_policy, delete_policy
+from app.schemas.rules import RequestObject, UpdateRequestObject
 from app.utils.write_rego import write_to_file
 
 app = FastAPI()
 
 
-@app.post("/save")
-async def write(rego_rule: RequestObject) -> dict:
+@app.post("/policy/save")
```

Comment:
Having only `/policies` would be more restful

Reply:
Okay, cool.